test/check_energy_and_forces.py

```diff
@@ -0,0 +1,71 @@
+import pickle
+import os
+import numpy as np
+import sys
+import multiprocessing as mp
+import argparse
+from ase.io import read, Trajectory
+from tqdm import tqdm
+
+def check_relaxed_forces(sid, path, thres):
+    """
+    Check all forces in the final frame of adslab is less than a threshold.
+    """
+    final_atoms = read(path)
+    forces = final_atoms.get_forces()
+    if not (np.max(np.abs(forces)) <= thres):
+        print(f"{sid} doesn't satisfy the force threshold")
+
+def check_adsorption_energy(sid, path, ref_energy, adsorption_energy):
+    final_energy = read(path)
+    if not abs((final_energy.get_potential_energy() - ref_energy) - adsorption_energy) < 1e-6:
+        print(f"{sid} doesn't satify energy equation")
+
+def check_DFT_energy(sid, path):
+    """
+    Given a relaxation trajectory, check to see if 1. final energy is less than the initial
+    energy, raise error if not. 2) If energies monotonically decrease in a trajectory.
+    If any frame(i+1) energy is slightly higher than frame(i) energy, flag it and plot the trajectory.
+    """
+    traj = Trajectory(path)
+    if traj[-1].get_potential_energy() > traj[0].get_potential_energy():
+        raise ValueError(f"{sid} has final DFT energy that's higher than the initial energy")
+    for idx, frame in enumerate(traj[:-1]):
+        next_frame = traj[idx+1]
+        diffE = next_frame.get_potential_energy() - frame.get_potential_energy()
```

Comment:
minor nit: could be consistent with naming style and use `diff_e`

Reply:
sounds good.